### ISSUE TITLE
Some questions about dataset.

### ISSUE THREAD

**Issue by Aatr0x13** [valentin-deschaintre/Single-Image-SVBRDF-Capture-rendering-loss#4]
Hi, 

Thanks for your inspiring works and the large-scale dataset. We are implementing a new method, which requires us to make a new dataset. I'm interested in data generation and have a few questions.
1. I reimplemented the renderer used in [Deschaintre et al. 2019] in PyTorch. And I find that sometimes the reflected light spot is larger compared to the images provided in [Deschaintre et al. 2018], or the light spot is outside the image, causing most areas of the image to be dark. Is this normal? Is the on-the-fly renderer equivalent to the Mitsuba renderer?
2. As described in the paper, the SVBRDF maps provided in [Deschaintre et al. 2018] are randomly scaled and rotated using Mitsuba, while the on-the-fly renderer only performs rendering. Also, you mentioned in another question that the visualization of the paper was generated using Mitsuba. Is it possible to provide the Mitsuba renderer with Cook-Torrance BRDF model and GGX normal distribution? This will certainly help us a lot.
3. When exporting SVBRDF maps in Substance Designer (I'm not sure if you did it in this way), are the normal and roughness maps exported in sRGB (or raw) space, while the diffuse and specular maps are exported in linear space? When only base color and metallic maps are provided, did you use the *basecolor_metallic_to_diffuse_specular* converter to get the diffuse and specular maps? These may be silly questions, but I'm still not 100% sure after a lot of Googling.

Computer graphics and material design are completely new to me and I hope you don't mind me asking so many questions. Thank you for your efforts and time!

Best,
Kakei

**Comment by valentin-deschaintre** [comment]
Hi thanks for your questions,

1. In 2019 I had a variety of options to increase the robustness to different flashes and flash directions which explains why you see different flash sizes for example. It is absolutely not the equivalent of Mitsuba as Mitsuba uses raytracing. In the context of this project, this renderer was sufficient to achieve good quality, but the trade-offs are yours to decide.
 
2. I am trying to find time to provide such implementation as a plugin to Mitsuba 2, but this will probably not happen in the next couple weeks. As it is it is added in the core of Mitsuba and requires recompiling and rediting specific file. It is not future proof at all and not trivial to share.

3. I did export maps from Substance Designer. I expect normal and roughness to be linear while albedos should be gamma corrected, but this is mostly dependent on how the artist designed its material. When only base color and metallic maps are provided, I used the node you mention. 
/!\ Please note that the Substance user agreement explicitly forbids its use to generate training data and that we asked for a special license to do so at the time.

I hope this answers your questions
Best,

**Comment by Aatr0x13** [comment]
Hi, thanks for your thorough reply, my questions are well-solved. 

I compared the results using Mitsuba's data and the data from the 2019 renderer and they are indeed similar, I think that's good enough for us, thanks again for your great work.

Thank you for the kind reminder, we will try to apply for a special license, any suggestions on who we should contact?

Best,
Kakei

**Comment by valentin-deschaintre** [comment]
Hi,

I believe for research purposes Tamy Boubekeur would probably be a good contact.

Best,
Valentin

**Comment by Aatr0x13** [comment]
Hi,

Thank you for your suggestion, we will contact Tamy Boubekeur immediately. 

Best,
Kakei
